docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -659,7 +659,7 @@ Note:
 
 |
 | event.values
-| An array of all cell values for the raw on which the action will execute.
+| An array of all cell values for the raw on which the action will execute. To access a column value, use `{{event.values.[x]}}`, where `x` represents the column number.
```

Comment:
Would you mind changing `raw` to `row`. Notice the `a` in the first vs `o`

Reply:
Another good catch! Yes — I'll fix that. Thank you!